modules/graphql/src/vulnerability.rs

```diff
@@ -27,6 +27,7 @@ impl VulnerabilityQuery {
                 published: vulnerability.vulnerability.published,
                 modified: vulnerability.vulnerability.modified,
                 withdrawn: vulnerability.vulnerability.withdrawn,
+                cwe: vulnerability.vulnerability.cwe,
```

Comment:
Do we know this is a 1-to-1 relationship? S'fine if so.

Reply:
To my understanding yes. But I might be wrong.